backend.py

```diff
@@ -280,38 +281,46 @@ def get_active_robots(self):
                 yield robot
 
 
-def get_start_tiles(board):
+def get_robot_names():
+    """
+    Return a list of robots names using pathlib.Path library.
```

Comment:
`pathlib.Path` je tady tzv. *implementační detail* – něco, co ta funkce používá, ale není to vidět "venku". Pro někoho, kdo tu funkci bude volat, to není důležité. Nemá proto cenu to zmiňovat v dokumentaci.
(Kdyby ta funkce vracela seznam cest jako `Path` objektů, bylo by to jinak, ale tady vrací seznam řetězců.)